requirements.txt

```diff
@@ -4,6 +4,7 @@ certifi==2025.10.5
 cheroot==11.0.0
 filelock==3.19.1
 isodate==0.7.2
+jaconv==0.4.0
```

Comment:
This also needs to be included in pyproject.toml dependencies for library users.